### PR TITLE
Added support for all Belgium national languages 🇧🇪

### PR DESCRIPTION
**Pull request by patrickallaert**
| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          |
| **Type**                                   | improvement
| **Target eZ Platform version** | ?
| **BC breaks**                          | no
| **Doc needed**                       | no

#### Checklist:
- [X] Provided PR description.
- [X] Tested the solution manually.
- [ ] Provided automated test coverage.
- [X] Checked that target branch is set correctly (master for features, the oldest supported for bugs).
- [X] Ran PHP CS Fixer for new PHP code (use `$ composer fix-cs`).
- [ ] Asked for a review (ping `@ezsystems/php-dev-team`).

### CODE REVIEW

File: eZ/Bundle/EzPublishCoreBundle/Resources/config/locale.yml [hunk 1]
```diff
@@ -56,7 +58,7 @@ parameters:
 
     ezpublish.locale.browser_map:
         au: ['eng-AU']
-        be: ['fre-BE']
+        be: ['fre-BE', 'dut-BE']
```
Comment (glye) [inline eZ/Bundle/EzPublishCoreBundle/Resources/config/locale.yml:61]: Shouldn't this also include the ger-BE you added?

Comment (patrickallaert) [reply eZ/Bundle/EzPublishCoreBundle/Resources/config/locale.yml:61]: Absolutely!

**Review by glye — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Review by webhdx — APPROVED** [review]

**Comment by webhdx** [comment]
@mikadamczyk Do you recall any other places where we should update languages map e.g. in AdminUI?

**Review by Steveb-p — APPROVED** [review]

**Comment by mikadamczyk** [comment]
> @mikadamczyk Do you recall any other places where we should update languages map e.g. in AdminUI?

@webhdx AFAIK no, the list of languages should always come from the kernel. We had one commit with such list (https://github.com/ezsystems/ezplatform-admin-ui/pull/770/commits/352a133d8b6bca6d9d0db5458635c530baf81b49) but it was removed before the merge

**Review by mikadamczyk — APPROVED** [review]

**Review by alongosz — APPROVED** [review]

**Comment by alongosz** [comment]
No further testing required. Merging.

**Comment by alongosz** [comment]
Thank you @patrickallaert :muscle:

**Pull request merged**